Player.js

```diff
@@ -0,0 +1,18 @@
+const PLAYER_VERSION = '0.1';
+
+class Player {
+  static getVersion() {
```

Comment:
Getters and setters are quite enterprisish and sometimes considered an anti-pattern. I personally dislike them. Since `PLAYER_VERSION` is constant, would it hurt to simply expose it as a field of Player and use it as such `res.json(Player.getVersion());` ➡️ `res.json(Player.version);`? Especially since this getter has no special logic, simply returns a field.